src/Shared/IdentityClient/Config/ConfigServicesIdentityService.cs

```diff
@@ -11,29 +15,29 @@ public static IServiceCollection IdentityApplicationBuilderExtensions(this IServ
             services
                 .AddAuthorization(options =>
                 {
-                    options.AddPolicy("admin", policy => policy.RequireClaim("Admin"));
+                    options.AddPolicy(UserPolicy.User, policy => policy.RequireAssertion(context => context.User.IsInRole("User")));
+                    options.AddPolicy(UserPolicy.Admin, policy => policy.RequireAssertion(context => context.User.IsInRole("Admin")));
                 })
                 .AddAuthentication(options => {
                     options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
                     options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
                 })
                 .AddJwtBearer(options =>
                 {
-                    options.Authority = "https://localhost:5123";
+                    options.Authority = "https://localhost:5001";

```

Comment:
`Authority` and `Audience `should be passed through configuration. (via appsettings.json). In production we will override values with environmental variables.